src/content/development/shipyard/settings/_index.en.md

```diff
@@ -28,28 +28,27 @@ The possible settings are:
   * **`cluster_count`**: Can be used to quickly deploy multiple clusters with an identical topology.
   * **`clusters`**: Map of clusters to deploy. Each key is the cluster name and the values are cluster specific settings.
 * Global and/or cluster specific:
-  * **`cni`**: Which CNI to deploy on the cluster, currently supports **`weave`** and **`ovn`**.
+  * **`cni`**: Which CNI to deploy on the cluster, currently supports the default kind CNI and **`ovn`**.
```

Comment:
Looks like the CNI is called [kindnet](https://github.com/aojea/kindnet)

Reply:
Yes; I chose not to mention it because we don’t actually specify it (we rely on the kind default), but on second thoughts it does seem worth mentioning because we know Submariner works on kindnet but if the kind default changes in future it may no longer work with whatever CNI replaces kindnet.